.github/workflows/alpha-release.yml

```diff
@@ -0,0 +1,42 @@
+name: 'Alpha Release'
+
+on:
+  schedule:
+    - cron: '0 20 * * 1' # weekly (Monday) 12 PM PST
+
+jobs:
+  test:
+    name: Test latest code
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: 12.x
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install dependencies for master
+        run: yarn install --frozen-lockfile --non-interactive
+      - name: Basic Tests
+        env:
+          CI: true
+          ASSERT_ALL_DEPRECATIONS: true
+        run: yarn test
+
+  release:
+    name: Run publish script
+    runs-on: ubuntu-latest
+    needs: [ test ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 12.x
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install dependencies for master
+        run: yarn install --frozen-lockfile --non-interactive
+      - name: publish with script
+        run: node bin/publish.js canary
```

Comment:
Don't we need to `--distTag` here? Also, what ensures that the version number includes `-alpha.0`?

Reply:
second question is not an issue:

https://github.com/emberjs/data/blob/fe844e484b0200f6ea0e8d7a5efdb79e81f7416a/bin/publish.js#L231-L236

Reply:
Hmm good point, let me figure out some code to follow: 

1. If current version is a minor version release such as `3.15.0`, the version picked for the next alpha is `3.16.0-alpha.0`
2. If current version is an alpha release such as `3.15.1-alpha.0`, the version picked for the next alpha is `3.15.1-alpha.1`
3. Beta releases would not affect the versioning of the next alpha version as the alpha will continue from the last alpha release.